packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -49,7 +49,7 @@ export const prodParasKusama: EndpointOption[] = [
     info: 'bifrost',
     homepage: 'https://ksm.vtoken.io/?ref=polkadotjs',
     paraId: 2001,
-    text: 'Bifrost (Kusama)',
+    text: 'Bifrost Kusama',
```

Comment:
Any reason why this is not simply “Bifrost” (this only affects the list display). The reason for asking is that it doesn’t s already under the Kusama tab. 

(Obviously the chain name later on is 100%)

Reply:
updated thx